robber/matchers/boolean.py

```diff
@@ -11,20 +11,15 @@ def matches(self):
         return self.actual is True
 
     def failure_message(self):
-        return 'Expected %s to be True' % self.actual
+        message = 'Expected {actual} to be {adj}'
 
+        if not self.is_negative:
+            adj = 'True'
+        else:
+            adj = 'False'
 
-class FalseMatcher(Base):
-    """
-    expect(false).to.be.false()
-    """
-
-    def matches(self):
-        return self.actual is False
-
-    def failure_message(self):
-        return 'Expected %s to be False' % self.actual
+        return message.format(actual=self.actual, adj=adj)
 
 
 expect.register('true', TrueMatcher)
-expect.register('false', FalseMatcher)
+expect.register('false', TrueMatcher, is_negative=True)
```

Comment:
I think the name `TrueMatcher` is quite confusing as this class also supports False cases...